src/sentry/static/sentry/app/views/usageStats/usageStatsProjects.tsx

```diff
@@ -1,65 +1,311 @@
+    return {
+      statsPeriod: dataDatetime?.period || DEFAULT_STATS_PERIOD,
+      interval: '1d',
+      groupBy: ['category', 'outcome', 'project'],
+      field: ['sum(quantity)'],
+    };
+  }
+
+  get tableMetaData() {
+    const {projectStats} = this.state;
+
+    return {
+      headers: this.tableHeader,
+      ...this.mapSeriesToTable(projectStats),
+    };
+  }
+
+  get tableSort(): {
+    key: SortBy;
+    direction: number;
+  } {
+    const {tableSort} = this.props;
+
+    if (!tableSort) {
+      return {
+        key: SortBy.TOTAL,
+        direction: 1,
+      };
+    }
+
+    let key: string = tableSort;
+    let direction: number = -1;
+
+    if (tableSort.charAt(0) === '-') {
+      key = key.slice(1);
+      direction = 1;
+    }
+
+    switch (key) {
+      case SortBy.PROJECT:
+      case SortBy.TOTAL:
+      case SortBy.ACCEPTED:
+      case SortBy.FILTERED:
+      case SortBy.DROPPED:
+        return {key, direction};
+      default:
+        return {key: SortBy.TOTAL, direction: -1};
+    }
+  }
+
+  get tableHeader() {
+    const {key, direction} = this.tableSort;
+
+    const getArrowDirection = (linkKey: SortBy) => {
+      if (linkKey !== key) {
+        return undefined;
+      }
+
+      return direction > 0 ? 'down' : 'up';
+    };
 
     return [
-      [
-        'orgStats',
-        `/organizations/${organization.slug}/stats_v2/projects/`,
-        {
-          query: {
-            interval: '1d',
-          },
-        },
-      ],
-    ];
+      {
+        key: SortBy.PROJECT,
+        title: t('Project'),
+        align: 'left',
+        direction: getArrowDirection(SortBy.PROJECT),
+        onClick: () => this.handleChangeSort(SortBy.PROJECT),
+      },
+      {
+        key: SortBy.TOTAL,
+        title: t('Total'),
+        align: 'right',
+        direction: getArrowDirection(SortBy.TOTAL),
+        onClick: () => this.handleChangeSort(SortBy.TOTAL),
+      },
+      {
+        key: SortBy.ACCEPTED,
+        title: t('Accepted'),
+        align: 'right',
+        direction: getArrowDirection(SortBy.ACCEPTED),
+        onClick: () => this.handleChangeSort(SortBy.ACCEPTED),
+      },
+      {
+        key: SortBy.FILTERED,
+        title: t('Filtered'),
+        align: 'right',
+        direction: getArrowDirection(SortBy.FILTERED),
+        onClick: () => this.handleChangeSort(SortBy.FILTERED),
+      },
+      {
+        key: SortBy.DROPPED,
+        title: t('Dropped'),
+        align: 'right',
+        direction: getArrowDirection(SortBy.DROPPED),
+        onClick: () => this.handleChangeSort(SortBy.DROPPED),
+      },
+    ].map(h => {
+      const Cell = h.key === SortBy.PROJECT ? CellProject : CellStat;
+
+      return (
+        <Cell key={h.key}>
+          <PseudoLink align={h.align as Alignments} onClick={h.onClick}>
+            {h.title}
+            {h.direction && <IconArrow size="xs" direction={h.direction as any} />}
+          </PseudoLink>
```

Comment:
You might be able to repurpose `app/components/gridEditable/sortLink` here. It does very similar things.